projects/angular/src/emphasis/alert/_alert.clarity.scss

```diff
@@ -186,7 +197,14 @@
   //close alert button.
   .alert-item > span,
   .alert-text {
-    display: inline-block; //needed for IE11
+    @include mixins.css-var(
+      display,
+      clr-link-display,
+      inline-block,
+      //needed for IE11
+      variables.$clr-use-custom-properties
+    );
+
```

Comment:
Do we need that and how it's connected with the `link` or the buttons?

Reply:
corrected in commit eb48c7f